aviary/subsystems/mass/gasp_based/fixed.py

```diff
@@ -578,19 +583,29 @@ def initialize(self):
 
     def setup(self):
         aviary_options: AviaryValues = self.options['aviary_options']
-
-        add_aviary_input(self, Aircraft.Engine.MASS_SPECIFIC, val=0.21366)
-        add_aviary_input(self, Aircraft.Engine.SCALED_SLS_THRUST, val=4000)
-        add_aviary_input(self, Aircraft.Nacelle.MASS_SPECIFIC, val=3)
-        add_aviary_input(self, Aircraft.Nacelle.SURFACE_AREA, val=5)
-        add_aviary_input(self, Aircraft.Engine.PYLON_FACTOR, val=1.25)
-        add_aviary_input(self, Aircraft.Engine.ADDITIONAL_MASS_FRACTION, val=0.14)
-        add_aviary_input(self, Aircraft.Engine.MASS_SCALER, val=1)
+        num_engine_type = len(aviary_options.get_val(Aircraft.Engine.NUM_ENGINES))
+        total_num_engines = aviary_options.get_val(Aircraft.Propulsion.TOTAL_NUM_ENGINES)
+
+        add_aviary_input(self, Aircraft.Engine.MASS_SPECIFIC,
+                         val=np.full(num_engine_type, 0.21366))
+        add_aviary_input(self, Aircraft.Engine.SCALED_SLS_THRUST,
+                         val=np.full(num_engine_type, 4000))
+        add_aviary_input(self, Aircraft.Nacelle.MASS_SPECIFIC,
+                         val=np.full(num_engine_type, 3))
+        add_aviary_input(self, Aircraft.Nacelle.SURFACE_AREA,
+                         val=np.full(num_engine_type, 5))
+        add_aviary_input(self, Aircraft.Engine.PYLON_FACTOR,
+                         val=np.full(num_engine_type, 1.25))
+        add_aviary_input(self, Aircraft.Engine.ADDITIONAL_MASS_FRACTION,
+                         val=np.full(num_engine_type, 0.14))
+        add_aviary_input(self, Aircraft.Engine.MASS_SCALER,
+                         val=np.full(num_engine_type, 1.0))
         add_aviary_input(self, Aircraft.Propulsion.MISC_MASS_SCALER, val=1)
-        add_aviary_input(self, Aircraft.Engine.WING_LOCATIONS, val=0.35)
+        add_aviary_input(self, Aircraft.Engine.WING_LOCATIONS,
+                         val=np.full(int(total_num_engines/2), 0.35))
 
-        self.add_input(
-            "main_gear_mass", val=500, units="lbm", desc="WMG: mass of main gear")
+        add_aviary_input(self, Aircraft.LandingGear.MAIN_GEAR_MASS,
+                         val=500, units="lbm", desc="WMG: mass of main gear")
```

Comment:
```suggestion
        add_aviary_input(self, Aircraft.LandingGear.MAIN_GEAR_MASS, val=500, units="lbm")
```
the description should get populated by add_aviary_input